sandbox.go

```diff
@@ -807,6 +808,14 @@ func OptionHostname(name string) SandboxOption {
 	}
 }
 
+// OptionSandboxName function returns an option setter for sandbox name option to
+// be passed to NewSandbox method.
+func OptionSandboxName(name string) SandboxOption {
+	return func(sb *sandbox) {
+		sb.config.name = name
+	}
+}
```

Comment:
Same here, unless we need the sandbox to carry the container name, we  do not need the Name option for this fix.